src/notfoundigest/handler-external.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { internalServerError, noContent } from '@adobe/spacecat-shared-http-utils';
+import { isObject } from '@adobe/spacecat-shared-utils';
+import { SlackClient, SLACK_TARGETS } from '@adobe/spacecat-shared-slack-client';
+import {
+  build404InitialSlackMessage,
+  build404SlackMessage,
+} from '../support/slack.js';
+import {
+  get404Backlink,
+  process404LatestAudit,
+} from '../support/notfound.js';
+import { getSlackContextForAlert, isConfigByOrgForAlert } from '../support/config.js';
+
+const ALERT_TYPE = '404';
+
+export default async function notFoundExternalDigestHandler(message, context) {
+  const { dataAccess, log } = context;
+
+  const organizations = await dataAccess.getOrganizations();
+  for (const organization of organizations) {
+    const orgConfig = organization.getConfig();
+    if (isObject(orgConfig)) {
+      const organizationId = organization.getId();
+      // eslint-disable-next-line no-await-in-loop
+      const sites = await dataAccess.getSitesByOrganizationIDWithLatestAudits(
+        organizationId,
+        ALERT_TYPE,
+        false,
+      );
+      if (sites.length > 0) {
+        const slackClient = SlackClient.createFrom(context, SLACK_TARGETS.ADOBE_EXTERNAL);
```

Comment:
Initial idea was to get the "target slack org" from the alert config. Now that we have two different handlers (one for internal, one for external) and different logic in each handler; I think we can remove the field in alert config. Any ideas? cc/ @solaris007

Reply:
yes, by "target slack org" I meant the slack workspace